setup/configuration/index.md

```diff
@@ -62,7 +66,33 @@ spring:
           token: [vault access token]
 ```
 
-For information about configuring other supported Config Server backends, see the [Spring Cloud Config Server documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.1.0.RELEASE/single/spring-cloud-config.html#_environment_repository).
+> The above example configures authentication for Vault using the Vault
+> `X-Config-Token` header, via the setting of the
+> `spring.cloud.config.server.vault.token` property. In Spinnaker 1.18, you  can
+> use other methods of configuring authentication for Vault. See the relevant
+> [Spring Cloud Config Server
+> documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.2.1.RELEASE/reference/html/#vault-backend)
+> for more information.
+
+### Using an S3 backend
+
+To use an AWS S3 backend, enable the `awss3` profile and configure the settings under `spring.cloud.config.server.awss3`. Your configuration might look like the following example:
+
+```yml
+spring:
+  profiles:
+    active: awss3
+  cloud:
+    config:
+      server:
+        awss3:
+          region: us-west-2
+          bucket: my-bucket
+```
+
+To configure authentication, you must use AWS's Default Credential Provider Chain. See the [AWS SDK documentation](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html) for more information.
+
+For information about configuring other supported Config Server backends, see the [Spring Cloud Config Server documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.2.1.RELEASE/single/spring-cloud-config.html#_environment_repository).
```

Comment:
```suggestion
For information about configuring other supported config-server backends, see the [Spring Cloud Config Server documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.2.1.RELEASE/single/spring-cloud-config.html#_environment_repository).
```

Reply:
I think I use "Config Server" generally as a shortened form of the project name (Spring Cloud Config Server), less as a generic term for something that serves configuration. Is that use OK?